src/mmrelay/matrix/e2ee_identity.py

```
@@ -0,0 +1,226 @@
+            _client_label(client, "device_id"),
+            exc,
+        )
+        logger.debug("Matrix cross-signing identity inspection failure", exc_info=True)
+        return None
+
+    if identity_property is not None and local_identity is None:
+        try:
+            server_has_identity = await _server_has_own_cross_signing_identity(client)
+        except asyncio.CancelledError:
+            raise
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)